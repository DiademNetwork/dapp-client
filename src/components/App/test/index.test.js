// Jest documentation https://facebook.github.io/jest/docs/en/getting-started.html
// Enzyme documentation http://airbnb.io/enzyme/docs/api/

import React from 'react'
import { shallow } from 'enzyme'
import { App } from '../'

describe('App', () => {
  it('Snapshot', () => {
    const component = shallow(<App />)
    expect(component).toMatchSnapshot()
  })
})
