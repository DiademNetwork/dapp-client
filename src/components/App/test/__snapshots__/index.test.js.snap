// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App
    message=""
    user="nobody"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <p>
          GetStream status: 
          pending
          <br />
          Message from contract: 
          <br />
          <br />
          <t
            appId="2107292709536080"
            autoLoad={true}
            callback={[Function]}
            cssClass="kep-login-facebook"
            fields="name,email,picture"
            size="metro"
            tag="button"
            textButton="Login with Facebook"
            typeButton="button"
          />
        </p>,
        <br />,
        <br />,
        "User logged: ",
        undefined,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "GetStream status: ",
            "pending",
            <br />,
            "Message from contract: ",
            undefined,
            <br />,
            <br />,
            <t
              appId="2107292709536080"
              autoLoad={true}
              callback={[Function]}
              cssClass="kep-login-facebook"
              fields="name,email,picture"
              size="metro"
              tag="button"
              textButton="Login with Facebook"
              typeButton="button"
            />,
          ],
        },
        "ref": null,
        "rendered": Array [
          "GetStream status: ",
          "pending",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          "Message from contract: ",
          undefined,
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "appId": "2107292709536080",
              "autoLoad": true,
              "callback": [Function],
              "cssClass": "kep-login-facebook",
              "fields": "name,email,picture",
              "size": "metro",
              "tag": "button",
              "textButton": "Login with Facebook",
              "typeButton": "button",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "br",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "br",
      },
      "User logged: ",
      undefined,
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <p>
            GetStream status: 
            pending
            <br />
            Message from contract: 
            <br />
            <br />
            <t
              appId="2107292709536080"
              autoLoad={true}
              callback={[Function]}
              cssClass="kep-login-facebook"
              fields="name,email,picture"
              size="metro"
              tag="button"
              textButton="Login with Facebook"
              typeButton="button"
            />
          </p>,
          <br />,
          <br />,
          "User logged: ",
          undefined,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "GetStream status: ",
              "pending",
              <br />,
              "Message from contract: ",
              undefined,
              <br />,
              <br />,
              <t
                appId="2107292709536080"
                autoLoad={true}
                callback={[Function]}
                cssClass="kep-login-facebook"
                fields="name,email,picture"
                size="metro"
                tag="button"
                textButton="Login with Facebook"
                typeButton="button"
              />,
            ],
          },
          "ref": null,
          "rendered": Array [
            "GetStream status: ",
            "pending",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "Message from contract: ",
            undefined,
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "appId": "2107292709536080",
                "autoLoad": true,
                "callback": [Function],
                "cssClass": "kep-login-facebook",
                "fields": "name,email,picture",
                "size": "metro",
                "tag": "button",
                "textButton": "Login with Facebook",
                "typeButton": "button",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
        "User logged: ",
        undefined,
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
